world-chain-builder/crates/world/rpc/src/eth.rs

```diff
@@ -83,6 +84,20 @@ where
     pub fn pool(&self) -> &Pool {
         &self.pool
     }
+
+    pub fn raw_tx_forwarder(&self) -> Option<&SequencerClient> {
+        self.sequencer_client.as_ref()
+    }
+
+    async fn maybe_forward_raw_transaction(&self, tx: Bytes, hash: B256) -> RpcResult<()> {
+        if let Some(client) = self.raw_tx_forwarder().as_ref() {
+            tracing::debug!( target: "rpc::eth",  "forwarding raw transaction to");
+            let _ = client.forward_raw_transaction(&tx).await.inspect_err(|err| {
+                    tracing::debug!(target: "rpc::eth", %err, hash=?*hash, "failed to forward raw transaction");
+                });
+        }
+        Ok(())
+    }
```

Comment:
Do we need a separate function for this? it seems minimal enough to put directly in `send_raw_transaction()`.

Reply:
It was just duplicate code in `send_raw_transaction `, and `send_raw_transaction_conditional` is why I separated it